### ISSUE TITLE
Logical `Gt`, `GtE`, etc.

### ISSUE THREAD

**Issue by rebcabin** [rebcabin/masr#38]
I began by restricting logical cmpops to `Eq` and `NEq`, but then discovered `LogicalCompare` expressions with `Gt`, `GtE` etc. in `expr13-10040d8`. I assume these are C-Like, in that `true == 1` is `Gt` `false == 0`. In any event, they're a little weird.

**Comment by certik** [comment]
Yeah, probably we can just support Eq and NEq. This is low priority.
